features/attached_enable.feature

```diff
@@ -657,11 +657,14 @@ Feature: Enable command behaviour when attached to an UA subscription
         And I will see the following on stdout
             """
             One moment, checking your subscription first
-            """
-        And I will see the following on stderr
```

Comment:
Is it ok that the error message ends up in `stdout` instead of `stderr`?

Reply:
I think for the user it will not matter that much. Or do you think we should keep that behavior somehow ?

Reply:
I have no strong opinion on this. I think we should keep errors in stderr just for conformance's sake, but I don't think it's mandatory

Reply:
I think this is a good point, but looking at the code, it seems that we have other places where we are printing errors on `stdout` instead of `stderr`.

For example:
https://github.com/canonical/ubuntu-advantage-client/blob/main/uaclient/cli.py#L1105
https://github.com/canonical/ubuntu-advantage-client/blob/main/uaclient/cli.py#L1127

And through the `action_enable` function.

Because of that, I think we could tackle this into a separate PR.
What do you all think @orndorffgrant @CalvoM @renanrodrigo ?

Reply:
No strong opinion here. I don't think anyone is relying on this message to be printed to stderr AFAIK

Reply:
Fair points, we can tackle it in the future as a maintenance task, lowest priority